workers/unity/Packages/io.improbable.gdk.tools/GdkToolsConfigurationProvider.cs

```diff
@@ -35,40 +37,36 @@ public class GdkToolsConfigurationWindow : EditorWindow
         private List<string> configErrors = new List<string>();
         private Vector2 scrollPosition;
 
-        // Minimum time required from last config change before saving to file
-        private readonly TimeSpan FileSavingInterval = TimeSpan.FromSeconds(1);
-        private DateTime lastSaveTime = DateTime.Now;
+        // Flag to indicate if we have unsaved changes in the settings window
         private bool hasUnsavedData;
 
-        [MenuItem("SpatialOS/GDK tools configuration", false, MenuPriorities.GdkToolsConfiguration)]
-        public static void ShowWindow()
+        public GdkToolsConfigurationProvider(string path, SettingsScope scope = SettingsScope.User)
+        : base(path, scope) { }
+
+        [SettingsProvider]
+        public static SettingsProvider CreateGdkToolsConfigurationProvider()
         {
-            GetWindow<GdkToolsConfigurationWindow>().Show();
+            var provider = new GdkToolsConfigurationProvider("Project/GDK Tools Configuration", SettingsScope.Project);
```

Comment:
Considering this is now in a `Project Settings` window, the `Configuration` bit might be echoing.
Should we renamed this to just `Project/Spatial OS` ? @jamiebrynes7 @paulbalaji @austinkelmore

Reply:
Yeah `Project/Spatial OS` might be a little cleaner :)

Reply:
👍